components/ood_classification.yaml

```diff
@@ -0,0 +1,86 @@
+name: OOD Classification
+description: |
+  OOD Classification
+inputs:
+  - {name: image, description: ''}
+  - {name: s3_bucket, description: ''}
+  - {name: owner, description: ''}
+  - {name: project, description: ''}
+  - {name: experiment, description: ''}
+  - {name: model, description: ''}
+  - {name: skip_tensorboard, description: ''}
+  - {name: s3_datadir, description: ''}
+  - {name: ood_data_train, description: ''}
+  - {name: ood_data_eval, description: ''}
```

Comment:
BTW why is it is necessary to have two set of incompatible parameters? Can't you use s3_datadir always?